### DISCUSSION TITLE
What's the meaning of project name "libdatachannel"?  Indeed I want to know the network stack of media transport?

### DISCUSSION THREAD

**Discussion by Romantic-LiXuefeng** [paullouisageneau/libdatachannel#819] [General]
@paullouisageneau Thanks for your great work!   I'm pretty new to the  "libdatachannel"?   From the project name, I guess the network stack(SCTP over DTLS and ICE) is same as the WebRTC "DataChannel" which designed to transfer raw data, either reliably or not.   

Maybe the main difference is that the "libdatachannel" used to transfer media data in reliably mode, don't need to implements the pacer flow and congestion control?   

`ICE <-> DTLS <-> SCTP  <-> MediaTransport(H264/opus)` 

Hope to give a clear conclusion!  If my guess is correct, I will deep into "libdatachannel" as this is my ideal plan for my project. Thanks again!

**Comment by paullouisageneau** [comment]
libdatachannel primarily implements WebRTC Datachannels (hence the name), but also has support for WebRTC media tracks, which are based on SRTP/SRTCP and run on the side, not over SCTP.

**Reply by Romantic-LiXuefeng** [reply]
oh! Thanks again
